js/src/dropdown.js

```diff
@@ -263,6 +242,30 @@ class Dropdown extends BaseComponent {
     })
   }
 
+  _completeHide(element, menu, relatedTarget) {
+    const hideEvent = EventHandler.trigger(element, EVENT_HIDE, relatedTarget)
```

Comment:
May be mistaken, but as I can see this is a private method.
could be changed to `_completeHide(relatedTarget)`
as `element = this._element` and `menu=this._menu`

Reply:
True :smile: